README.md

```diff
@@ -69,6 +69,17 @@ cadenzaClient.show('{embeddingTargetId}', {
 - If the embedding target cannot be resolved, a 404 page is shown to the user.
 - Cadenza JS does not handle user authentication: If the user is not already logged in, the normal authentication flow of Cadenza will run. By default, the login page would be shown to the user.
 
+#### Show the Generated PDF of a Jasper Report View Directly
+Views of type "JasperReports report" can be shown in an iFrame like any other view.
```

Comment:
Isn't "JasperReports report" a bit redundant? -> "Jasper Report"

Reply:
To me "JasperReports" is just a product name, but I'm fine with your version, too. I'll change it.

Reply:
Now I remember why I wrote it like that. "JasperReports report" is what the visualization type is called in Cadenza.

Reply:
OK :shrug: